Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -3189,7 +3189,7 @@ def test_skip(self):
         return self.runStep()
 
 
-class TestCheckPatchRelevance(BuildStepMixinAdditions, unittest.TestCase):
+class TestCheckChangeRelevance(BuildStepMixinAdditions, unittest.TestCase):
```

Comment:
Should add some unit-test for PR as well. Especially after my above comment for detecting PR vs patch.